book/03-git-branching/sections/basic-branching-and-merging.asc

```diff
@@ -44,8 +45,10 @@ $ git checkout iss53
 .Creating a new branch pointer
 image::images/basic-branching-2.png[Creating a new branch pointer.]
 
-You work on your website and do some commits.
-Doing so moves the `iss53` branch forward, because you have it checked out (that is, your `HEAD` is pointing to it):
+You work on your website and do some commits. Doing so moves the `iss53`
+branch forward (but not the `master` branch), because it's the `iss53`
+branch that you have checked out; that is, your `HEAD` is now pointing to
+the commit at the tip of the `iss53` branch:
```

Comment:
Line breaks again. Also, you have a fact wrong: `HEAD` points to the `iss53` branch, and _that_ branch points to a commit. When `HEAD` points to a commit we call that a "detached head" state.

Let's remove the parenthetical. When we say that committing moves `iss53` forward, we don't mention all the things that committing _doesn't_ do. It doesn't, for example, change anything in your configuration. The fact that `master` doesn't move is also illustrated in the figure just below:

![image](https://user-images.githubusercontent.com/39902/30776876-4b830e54-a063-11e7-82c8-014108b8e549.png)

Reply:
I'll just revert this to the original text.